pints/_core.py

```diff
@@ -341,3 +341,28 @@ def values(self):
         points and ``n_outputs`` is the number of outputs.
         """
         return self._values
+
+
+class TunableMethod(object):
+
```

Comment:
Extra newline!

Reply:
where? the space after the class definition is put there by my auto-formatter (autopep8)